### PR TITLE
RE-1976 Fix dual schedules

### PR DESCRIPTION
**Pull request by hughsaunders**
Two timer triggers were added to some jobs in order to avoid
the maintenance window. However only one makes it into the
Jenkins configuration. This commit switches from two timer
triggers to a single timer trigger that contains two cron
expressions.

Issue: [RE-1976](https://rpc-openstack.atlassian.net/browse/RE-1976)

### CODE REVIEW

File: scripts/extract_dsl.py [hunk 1]
```diff
@@ -29,7 +29,6 @@ def extract_dsl(jjbfile, outdir):
                 item=item[key]['name'])
             with open(outfile, "w") as outf:
                 outf.write(dsl)
-            print("Written {outfile}".format(outfile=outfile))
```
Comment (odyssey4me) [inline scripts/extract_dsl.py]: Why was this removed?

Comment (hughsaunders) [reply scripts/extract_dsl.py]: log spam
